common/server_card.cpp

```diff
@@ -47,15 +57,25 @@ void Server_Card::resetState()
     setPT(QString());
     setAnnotation(QString());
     setDoesntUntap(false);
+    setFaceDown(false);
```

Comment:
this causes a major bug: cards have their state reset when moved between the battlefield and the deck, their facedown state is then checked afterwards to determine what event to show to other players. with this change moving a facedown card to your deck (unknown to unknown) will tell all your opponents (but not you) what card it was.

Reply:
Oh, right. I think I changed that (or intended to anyways) in my refactored implementation of `::moveCard` in the other PR, but not here. Will revert

Reply:
Right, I believe I changed this in the other PR but not there… Reverting